packages/SwingSet/src/kernel/kernel.js

```diff
@@ -435,6 +435,11 @@ export default function buildKernel(
       // eslint-disable-next-line no-use-before-define
       const deliveryResult = await vatWarehouse.deliverToVat(vatID, kd, vd, vs);
       insistVatDeliveryResult(deliveryResult);
+      if (vd[0] !== 'bringOutYourDead') {
```

Comment:
This includes gcActions too, right? That's probably the right thing to do, but I wanted to be conscious about it. If we have gcActions to give to a vat, we should process them before reaping/boyding, in the hopes that a `dispatch.dropExport` will free up more garbage.

Reply:
Yes.  All dispatches funnel through `deliverAndLogToVat`.